modules/speex/speex.c

```diff
@@ -437,6 +441,10 @@ static void config_parse(struct conf *conf)
 		sconf.complexity = v;
 	if (0 == conf_get_u32(conf, "speex_enhancement", &v))
 		sconf.enhancement = v;
+	if (0 == conf_get_u32(conf, "speex_mode_nb", &v))
+		sconf.vbr = v;
+	if (0 == conf_get_u32(conf, "speex_mode_wb", &v))
+		sconf.vbr = v;
```

Comment:
why is the .vbr variable set for "speex_mode_nb" param?

should it not be .mode_nb and .mode_wb instead?